docs/java-rest/high-level/document/update.asciidoc

```diff
@@ -17,7 +17,6 @@ An +{request}+ requires the following arguments:
 include-tagged::{doc-tests-file}[{api}-request]
 --------------------------------------------------
 <1> Index
-<2> Type
 <3> Document id
 
```

Comment:
`<2> Document id` instead of  `<3> Document id>`?

Reply:
👍